-- Switch from extlib_qpdf_v12.cmake to extlib_qpdf_v11.cmake because
-- FreeBSD's print/qpdf port is still at the v11.x API/ABI.
--- CMakeLists.txt.orig	2026-10-05 07:00:27 UTC
+++ CMakeLists.txt
@@ -136,8 +136,8 @@ include(cmake/extlib_lcms2.cmake)
 include(cmake/extlib_jpeg.cmake)
 include(cmake/extlib_openjpeg.cmake)
 include(cmake/extlib_lcms2.cmake)
-# include(cmake/extlib_qpdf_v11.cmake)
-include(cmake/extlib_qpdf_v12.cmake)
+include(cmake/extlib_qpdf_v11.cmake)
+# # include(cmake/extlib_qpdf_v12.cmake)
 include(cmake/extlib_blend2d.cmake)
 include(cmake/extlib_freetype.cmake)
 include(cmake/extlib_pdfium_jbig2.cmake)
